src/pages/home/report/ReactionList/PopoverReactionList.js

```diff
@@ -182,12 +101,13 @@ class PopoverReactionList extends React.Component {
                 emojiCount: 0,
             };
         }
-        const emojiCount = selectedReaction.users.length;
-        const reactionUsers = _.map(selectedReaction.users, (sender) => sender.accountID);
-        const emoji = EmojiUtils.findEmojiByName(selectedReaction.emoji);
+        const reactionUsers = _.pick(selectedReaction.users, _.identity);
+        const emojiCount = _.map(reactionUsers, (user) => user).length;
+        const userAccountIDs = _.map(reactionUsers, (user, accountID) => parseInt(accountID, 10));
+        const emoji = _.find(emojis, (e) => e.name === selectedReaction.emojiName);
```

Comment:
I think we might not want to use emoji here, and instead use `EmojiUtils.findEmojiByName` because we are also supporting spanish

Reply:
Ok, done